aut_docs/Installation_Guide_WindowsOS.md

```diff
@@ -0,0 +1,230 @@
+## Installation Guide for Windows OS
+
+### A one-time setup:
+
+- Download [Python 3.6.5](https://www.python.org/downloads/release/python-365) (Windows x86-64 executable installer version 3.6.5) 
+  
+  **Note:** Don't forget to check **Add to path** while installing.
+
+- Check the python version 
+    ```
+       python --version
+    ```
+
+- Check if pip (The python package installer) is already installed by running
+    ```
+       pip --version
+    ```
+- If installed then upgrade it using the command
+    ```
+       python -m pip install --upgrade pip
+    ```
+  This will upgrade the pip version
+
+- Install virtual environment
+    ```
+       pip install virtualenv
+    ```
+- Download [Git](https://git-scm.com/download/win) 
+  
+  **Note:** While installing select the option **Use Git and optional Unix Tools from the Windows Command Prompt** for using Git from the command prompt
+ 
+- Clone the VMS project
+    ```
+       git clone https://github.com/systers/vms.git
+    ```
+
+- Create python3.6 virtual environment
+    ```
+       virtualenv -p python3.6 venv
+    ```
+    
+### To start a testing session:
+
+- Activate virtual environment
+    ```
+       .\venv\Scripts\activate
+    ```
+    
+- Change directory to VMS code
+    ```
+       cd vms\vms
+    ```
+
+- Install all python dependencies
+    ```
+       pip install -r requirements.txt
+    ```
+
+### Install PostgreSQL
+
+- If you are installing and running the project on your local machine, then you will need to download [PostgreSQL](http://www.postgresql.org/download/) (version >= 9.3.4) and install it (Remember the password that you have entered during the installation process):
+  
+  **Note:** Add bin & lib to path, Example: C:\Program Files\PostgreSQl\9.5\bin & C:\Program Files\PostgreSQl\9.5\lib
+
+- Download [Firefox](https://www.mozilla.org/en-US/firefox/new/) web browser
+  
+  **Note:** Use Firefox (version 60)
+
+### Setup PostgreSQL
+
+- To start the postgreSQL Server there are 3 ways
+
+## Command Prompt
+    ```
+       pg_ctl -D "C:\Program Files\PostgreSQL\9.5\data" start
+    ```
```

Comment:
This appears not formatted correctly in the preview.

Reply:
ok I will check the preview

Reply:
It is beacause of the ## Command Prompt I will change it